packages/orchestrator-ui-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@orchestrator-ui/orchestrator-ui-components",
-    "version": "5.0.0",
+    "version": "6.0.0",
```

Comment:
With a major version bump we need to explain the consumer how to make changes in their app to make it work again -- discussion if needed